src/types/Notification.ts

```diff
@@ -2,10 +2,10 @@ export default interface Notification {
   id: string
   type: string
   reason?: string | null
-  created: number
+  created: string | Date
```

Comment:
```suggestion
  created: Date
```
Let's be strict about the type.